pkg/storage/sasNode.go

```diff
@@ -31,9 +31,12 @@ import (
 	"github.com/pkg/errors"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
-	"k8s.io/klog"
+	"k8s.io/klog/v2"
 )
 
+var globalRemovedDevicesMap = map[string]string{}
```

Comment:
It's not a blocker but if you have time, change this map so it can hold a timestamp recording when the volume is unpublished, so you can prune it from the list after 5 minutes or so has elapsed.